app/src/main/java/org/mozilla/fenix/collections/Tab.kt

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.collections
+
+import android.graphics.Bitmap
+
```

Comment:
do we need any kdocs here?

Reply:
It would be nice, but I made the conscious decision to not add it to keep the refactor simple. I will leave it as a follow up as we do more work here.